drone_flightplan/create_flightplan.py

```diff
@@ -84,7 +91,7 @@ def main():
     group.add_argument(
         "--gsd",
         type=float,
-        help="The flight altitude in meters.",
+        help="The ground sampling distance in meters.",
```

Comment:
Thanks for the correction here! Units are actually `cm/px` (centimetres per pixel)

Reply:
Thanks! I've corrected the unit measurements